test/generators/counterfactuals.jl

```diff
@@ -0,0 +1,142 @@
+include("feature_tweak.jl")
+include("generators.jl")
+include("probe.jl")
```

Comment:
Aren't we supposed to add those include/using statements in the `runtests.jl` file?

Reply:
We can do it both ways, but my current structure is that there's one main test file in each separate folder that is included in the `runtests.jl` file and this main file includes other test files from the same folder. I'm fine with including all of them in the `runtests.jl` file as well if you prefer that, it's just that the list of files there would get rather long.